int32 roboyID
float32[] voltage